transaction/src/plan.rs

```diff
@@ -40,6 +41,8 @@ pub struct TransactionPlan {
     pub fee: Fee,
     pub clue_plans: Vec<CluePlan>,
     pub memo_plan: Option<MemoPlan>,
+    pub valid_before: Timestamp,
```

Comment:
Note that the transaction crate does not depend on tendermint, only IBC, and so has to use the IBC `Timestamp` type.